server.conf.in

```diff
@@ -127,6 +127,25 @@
 # Defaults to firefox
 # browser = firefox
 
+# Path to the Selenium driver to use for recordings.
+# If set the driver must match the browser being used (for example,
+# "/usr/bin/geckodriver" for "firefox"). If no driver is explicitly set Selenium
+# Manager will try to find the right one in $PATH, downloading it as a fallback.
+# Note that Selenium Manager does not work in some architectures (for example,
+# Linux on arm64/aarch64), so in those architectures the driver must be
+# explicitly set.
+#driver =
+
+# Path to the Selenium browser binaries to use for recordings.
+# If set the binaries must match the browser being used (for example,
+# "/usr/bin/firefox-esr" for "firefox"). If not explicitly set Selenium
+# Manager will try to find the right binaries in $PATH, downloading it as a
+# fallback.
+# Note that Selenium Manager does not work in some architectures (for example,
+# Linux on arm64/aarch64), so in those architectures the browser binaries must
+# be explicitly set.
+#binaryLocation =
```

Comment:
Although the Selenium option is indeed `binary_location` in this context I think it would be better to name the option `browserPath`, for consistency with `browser` and the mentioned `driverPath`.